tests/unit/ai/test_secrets.py

```
@@ -0,0 +1,277 @@
+        "token",
+        "aws-access-key",
+        "aws-secret-key",
+        "github-pat",
+        "openai-key",
+        "rsa-private-key",
+        "ec-private-key",
+        "generic-private-key",
+    ],
+)
+def test_detects_secret_pattern(
+    description: str,
```

Ruff (0.15.6)

[warning] Unused function argument: `description`

(ARG001)